sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy

```diff
@@ -210,4 +221,12 @@ class ProjectOverviewView extends VerticalLayout{
             }
         }
     }
+
+    private void subscribeToProject(ProjectSummary project) {
```

Comment:
You could also provide just the project code here since this is the only thing you need here
```suggestion
    private void subscribeToProject(String projectCode) {
```

Reply:
Good Idea 👍